test/server/cards/02_SHD/upgrades/ArmedToTheTeeth.spec.ts

```diff
@@ -0,0 +1,35 @@
+describe('Armed to the teeth upgrade', function () {
+    integration(function (contextRef) {
+        describe('Armed to the teeth\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [{ card: 'bokatan-kryze#fighting-for-mandalore', upgrades: ['armed-to-the-teeth'] }],
+                        spaceArena: ['collections-starhopper'],
+                        base: { card: 'echo-base', damage: 10 }
```

Comment:
Can we remove any test settings that aren't relevant to the tests? Seems like setting the bases isn't really needed